packages/convenient_test_common_dart/lib/src/consts.dart

```diff
@@ -1,8 +1,21 @@
-const kConvenientTestManagerHost = '127.0.0.1';
-const kConvenientTestManagerPort = 3579;
+class SharedConfig {
+  static String getConfigFromEnv(String key, String defaultVal, {bool required = false}) {
+    final value = String.fromEnvironment(key);
 
-const kWorkerVmServiceHost = '127.0.0.1';
-// TODO temporarily hardcode this
-const kWorkerVmServicePort = 9753;
+    if (value.isEmpty && required) {
+      throw Exception('Please provide `$key` via --dart-define');
+    }
 
-const kReportFileExtension = 'bin';
+    return value.isEmpty ? defaultVal : value;
+  }
+
+  static String get kConvenientTestManagerHost => getConfigFromEnv('MANAGER_HOST', '127.0.0.1');
```

Comment:
nit: Shall we use `state late final ... = ...;`, b/c getters are slow (need to compute again whenever called)